examples/example-emailtask/nodes/email.json

```
@@ -0,0 +1,29 @@
+{
+  "id": "48286e12-ba6c-4627-97fd-2296a20c9db1",
+  "name": "Send Email",
+  "description": "Send e-mail.",
+  "deserializeAs": "EmailTask",
+  "inputVariables": [
+    {
+      "key": "exampleEmailTo",
+      "type": "PROCESS"
+    },
+    {
+      "key": "timestamp",
+      "type": "PROCESS"
+    },
+    {
+      "key": "tenantId",
+      "type": "PROCESS"
+    }
```

```suggestion
    }
```
I don't think the `timestamp` and `tenantId` are needed.